gubernator/main.py

```diff
@@ -64,12 +64,15 @@ def get(self):
         self.render('config.html', {'hostname': hostname})
 
     def post(self):
+        self.check_csrf()
```

Comment:
Does this check need to be performed for other request types or by other handlers?

Reply:
No, it's just for sensitive POST pages like this. There are no other POSTable pages that this would affect.